### PR TITLE
opencv: update to 3.1.0

### PR DESCRIPTION
**Pull request by lijpsummer**
Signed-off-by: lijpsummer <summer@wrtnode.com>

### CODE REVIEW

File: libs/opencv/Makefile [hunk 1]
```diff
@@ -9,12 +9,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=opencv
-PKG_VERSION:=3.0.0
-PKG_RELEASE:=1
+PKG_VERSION:=3.1.0
+PKG_RELEASE:=2
```
Comment (champtar) [inline libs/opencv/Makefile]: PKG_RELEASE needs to stay at 1 as you are increasing PKG_VERSION

**Comment by champtar** [comment]
sorry for the delay, can you just change PKG_RELEASE

**Comment by champtar** [comment]
@lijpsummer

**Comment by acinonyx** [comment]
That's weird... The MD5 sum from sourceforge is different.

**Comment by champtar** [comment]
i've just merged #2090
please open another PR to change PKG_SOURCE_URL

**Pull request closed without merging**